modules/.gitignore

```diff
@@ -15,6 +15,8 @@ lib/
 liferay-hook.xml.processed
 node_modules/
 npm-debug.log
+!**/resources/lib
```

Comment:
Is this specifically for things in `src/main/resources`?  If so, lets be explicit, the `**` is too broad.

Reply:
I'll look into this